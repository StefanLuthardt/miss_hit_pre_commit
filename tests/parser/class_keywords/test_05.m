% (c) Copyright 2021 Florian Schanda

classdef test_05
    methods

        function e = end()
            e = 0;
        end

    end
end
