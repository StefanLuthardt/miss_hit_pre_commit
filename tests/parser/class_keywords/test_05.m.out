----------------------------------------------------------------------
--  Parse tree for test_05.m
Root: Class_File
  Classdef: Class_Definition
    Name: Identifier <test_05>
    Methods: Methods Special_Block
      Items: Function_Definition
        Signature: Function_Signature
          Name: Identifier <end>
          Outputs: Identifier <e>
        Body: Sequence_Of_Statements
          Statements: Simple_Assignment_Statement
            LHS: Identifier <e>
            RHS: Number_Literal <0>
----------------------------------------------------------------------
MISS_HIT Debug Summary: 1 file(s) analysed, everything seems fine
